python_modules/dagster/dagster/_core/definitions/asset_check_factories/utils.py

```diff
@@ -262,3 +272,27 @@ def inner(fn: MultiAssetCheckFunction) -> AssetChecksDefinition:
         )(fn)
 
     return inner
+
+
+def build_multi_asset_check_factory(
```

Comment:
```suggestion
def build_multi_asset_check(
```

It's building a multi-asset check, not a factory, right?